firebase-firestore/src/main/java/com/google/firebase/firestore/local/MutationQueue.java

```diff
@@ -25,6 +25,9 @@
 
 /** A queue of mutations to apply to the remote store. */
 interface MutationQueue {
+  /** The tag used by the StatsCollector. */
+  String TAG = "Mutations";
```

Comment:
It would be nice if these tags matched the table names we're reading from. This one would be `mutations` and the other one `remote_documents`.